src/Testcontainers.EventHubs/Testcontainers.EventHubs.csproj

```
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFrameworks>net6.0;net8.0;netstandard2.0;netstandard2.1</TargetFrameworks>
```

```suggestion
        <TargetFrameworks>net8.0;net9.0;netstandard2.0;netstandard2.1</TargetFrameworks>
```

We need to merge `main` and add the test project to the [CI](https://github.com/testcontainers/testcontainers-dotnet/blob/60f50b6f608a5f2c3eebfbcd1f84a2a693f20a89/.github/workflows/cicd.yml#L34-L84) pipeline.